js/coinfloor.js

```diff
@@ -317,7 +316,7 @@ module.exports = class coinfloor extends Exchange {
             };
         }
         return {
-            'id': id,
+            'id': undefined,
```

Comment:
@kroitor why discard the id?

Reply:
Because it was not unique for two ledger entries )